pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1673,6 +1674,11 @@ type ObjectZoneSpec struct {
 	// +nullable
 	// +optional
 	CustomEndpoints []string `json:"customEndpoints,omitempty"`
+
+	// Preserve pools on object zone deletion
+	// +optional
+	// +kubebuilder:default:=true
+	PreservePoolsOnDelete bool `json:"preservePoolsOnDelete,omitempty"`
```

Comment:
Let's not set `omitempty` here so we can always see the value when describing the resource.

```suggestion
	PreservePoolsOnDelete bool `json:"preservePoolsOnDelete"`
```